README.rst

```diff
@@ -279,6 +279,12 @@ List of supported endpoints
     # AI Generated Photos
     amadeus.media.files.generated_photos.get(category='MOUNTAIN')
 
+    # Flight Offers Price
+    flights = amadeus.shopping.flight_offers_search.get(originLocationCode='SYD', destinationLocationCode='BKK', departureDate='2020-05-01', adults=1).data
+    amadeus.shopping.flight_offers.pricing.post(flights[0])
+    amadeus.shopping.flight_offers.pricing.post(flights[0:2], include='credit-card-fees,other-services')
```

Comment:
POST should receive a JSON body. I would suggest to pass the whole output of Flight Offers Search and implement the indexation within the post method implementation.